andrea/js/app.js

```diff
@@ -0,0 +1,74 @@
+// buttons //
+let signUpButton = document.querySelector("#navbar-sign");
+let logInButton = document.querySelector("#navbar-login");
+
+let createButton = document.querySelector(".create-btn")
+
+let welcomeView = document.querySelector(".welcome-view")
+
+// form //
+let form = document.querySelector(".register-form");
+let registerTitle = document.querySelector(".register-title");
+let logInTitle = document.querySelector(".login-title");
+let registerFields = document.querySelectorAll(".register-fields");
+
+let logInForm = document.querySelector(".login-form");
+let formAlternative = document.querySelector(".form-alternative");
+
+// passwords //
+let password = document.getElementById("password");
+let confirmPassword = document.getElementById("confirm-password");
+
+// show register form //
+function showRegister() {
+    welcomeView.classList.add("hidden");
+    signUpButton.classList.add("hidden");
+    logInButton.classList.add("hidden");
+    form.classList.add("form__show");
+    registerTitle.classList.remove("hidden");
+    signUpButton.removeEventListener("click", showRegister);
+}
+
+signUpButton.addEventListener("click", showRegister);
+
+// show log in form //
+function showLogin() {
+    form.classList.add("form__show");
+    registerTitle.classList.add("hidden");
+    signUpButton.classList.add("hidden");
+    logInButton.classList.add("hidden");
+    createButton.classList.add("hidden");
+    welcomeView.classList.add("hidden");
```

Comment:
As you can see in your code the `...classList.add(STRING)` and the `.remove` is repeated too many times, which goes against the DRY principle. Can you think of a way to extract these 2 commonly used methods into a reusable function? Maybe something like `function visibilityToggle(element, method, classname) { method === "add" ? element.classList.add(classname) : element.classList.remove(classname)`